tools/testing/kata-webhook/webhook-check.sh

```diff
@@ -16,6 +16,7 @@ source "${webhook_dir}/common.bash"
 
 readonly hello_pod="hello-kata-webhook"
 # The Pod RuntimeClassName for Kata Containers.
+RUNTIME_CLASS=$(kubectl get configmap kata-webhook -o jsonpath='{.data.runtime_class}')
 RUNTIME_CLASS="${RUNTIME_CLASS:-"kata"}"
```

Comment:
We should remove this since it's covered by line 19.

Reply:
Agree. Either the runtime class will be defined in the configmap and get picked up on L19, or it won't and the webhook pod will just fail to come up; assigning a default won't help. Maybe if we can't find the runtime class inside the configmap, we can fail the check earlier and hint at the user that they are missing that?

Reply:
On second thought, we shouldn't break the current behavior that allows passing an environment variable. Let's do it this way: if the env variable is passed, use that, otherwise fall back to querying the configMap.

Reply:
I can remove it and put it all in one line, but two lines keep readability better. 

As for using the default, the `main.go` file looks for the environment variable set by the configmap and uses it to set the runtimeclass, otherwise the code already sets the default runtime to "kata", but the pod being mutated will not fail to come up with the current code.

Reply:
I have updated the command to use the env variable runtime class if specified locally, otherwise check the configmap, and finally use "kata" as the default.